packages/Datadog.Unity/Runtime/DdUnityLogHandler.cs

```diff
@@ -44,7 +47,12 @@ public void LogException(Exception exception, UnityEngine.Object context)
         {
             try
             {
-                _ddLogger.Critical(exception.Message, error: exception);
+                _rum?.AddError(exception, RumErrorSource.Source);
```

Comment:
this means that if RUM is not enabled, then nothing will be logged? I think it may be confusing for the customer.

Let's maybe keep the existing behavior? Yes, it is not ideal, there won't be the link, but at least customer will be sure that if they logged exception with logger, they will find it in Logs.

Reply:
Confirmed with Product only logging to RUM is the correct behavior for now.